Gemfile

```diff
@@ -103,6 +103,27 @@ gem 'rdf-normalize', '~> 0.5'
 
 gem 'private_address_check', '~> 0.5'
 
+group :opentelemetry do
+  gem 'opentelemetry-exporter-otlp', '~> 0.26.3', require: false
```

Comment:
Super-minor style thing here -- since every single gem in this group needs the option, we could put these lines into a `with_options require: false` block. Might reduce some noise and make every line marginally more readable with just the gem and version name.

Reply:
I dont think you can use `with_options` in the Gemfile, because this is an Active Support method and it is not yet loaded

Reply:
Oh right, duh. You are correct.